pipeline/ingest/histology.py

```diff
@@ -467,16 +467,18 @@ def _search_histology_files(self, file_type):
             'corresponding_shanks': corresponding_shanks
         }
 
-        if (self.directory / 'channels.rawInd.npy').exists():
+        chan_loc_file = '{}_{}_{}_channel_locations.json'.format(
+            self.water, self.session_date_str, self.probe)
+
+        if (self.directory / chan_loc_file).exists():
 
             res['format'] = 2
 
             log.debug('format 2 detected..')
 
             new_histology_files = [
+                (self.directory / chan_loc_file),
                 (self.directory / 'channels.rawInd.npy'),
```

Comment:
@ixcat do you mean to also delete this line?

Reply:
no, want to include all 3 if available but just key on chan_loc_file if this is 'version2-like' -
that said, should update the return value based on 'new_histology_files_state' so will do this

Reply:
oh sorry I meant the line below, the `(self.directory / 'channels.rawInd.npy')`, as we discussed not to have to rely on the `channels.rawInd.npy` file to detect `format-2`

Reply:
ok - had misadjusted the file sequence & was returning all filenames irrespective of if they existed or not -
readjusted, now we check for the json as required and as the indicator for version2 and return all available version2 files (but the rest are not currently used)